paths/intro-to-github/03-exploring-repository.md

```diff
@@ -30,8 +30,8 @@ tell-me-why: |
 
   - **Projects:** You can create Kanban style project tracking boards within GitHub.
   - **Wiki**: GitHub also has a place to create and store your relevant project documentation.
-  - **Pulse**: Pulse is our project's dashboard. It contains information on the work that has been completed and the work in progress.
-  - **Graphs**: Graphs provide a more granular view of the repository activity, including who has contributed, when the work has been done, and who has forked the repository.
+  - **Pulse**: Pulse is our project's dashboard. It contains information on the work that has been completed and the work in progress and can be found under the Insights menu option.
```

Comment:
@beardofedu I realize this is already merged, but should we call this the Insights tab to keep things consistent down the page (e.g. Navigate to the Code tab on line 15)